docs/website/sidebars.js

```diff
@@ -33,6 +33,13 @@ const sidebars = {
       label: 'For Developers',
       items: [
         'for-developers/index',
+        {
```

Comment:
I suspect that we'll end up with several tens of these terms.

So I propose a single `Glossary.md` file, in which people can use CTRL-F to find something. And hopefully HTML anchors will let us link to specific definitions within that file.

Reply:
Good point.  I'll add that file. The only problem is that linking to files not in the sidebar makes it disappear. So I'll see if I can figure out a way to solve this.

Reply:
This is not great because people would need one extra click to access the index, but it's the only option I could find.

![image](https://github.com/input-output-hk/ouroboros-consensus/assets/175315/f726aa6c-5e7b-4367-8524-3490a559826c)